tableaudocumentapi/field.py

```diff
@@ -71,6 +65,53 @@ def name(self):
 
         return self.id
 
+    @property
+    def id(self):
+        """ Name of the field as specified in the file, usually surrounded by [ ] """
+        return self._id
+
+    @property
+    def caption(self):
+        """ Name of the field as displayed in Tableau unless an aliases is defined """
+        return self._caption
+
+    @property
+    def alias(self):
+        """ Name of the field as displayed in Tableau if the default name isn't wanted """
+        return self._alias
+
+    @property
+    def datatype(self):
+        """ Type of the field within Tableau (string, integer, etc) """
+        return self._datatype
+
+    @property
+    def role(self):
+        """ Dimension or Measure """
+        return self._role
+
+    @property
+    def is_quantitative(self):
+        return self._type == 'quantitative'
```

Comment:
I like this approach, it would allow for some really clean filtering `[f for f in fields if f.is_quantitative]`